app/const.py

```diff
@@ -302,3 +302,7 @@
         'url_regexp': 'https?://(www\\.)?linkedin\\.com/.*',
     },
 ]
+
+# The threshold for email notification to an admin user. When the number of
+# unreviewed notes is greater than this value, PF will send email notification.
+DEFAULT_THRES_NUM_UNREVIEWED_NOTES = 5
```

Comment:
Use clearer name here as well. Also, please add a line break after this line. Text files should end with a line break unless you have special reason.